group-community.md

```diff
@@ -28,45 +28,50 @@ We could use a network of hospex contacts (To Be Done) to perform the same searc
 
 - We host people in our friends network, so we can feel safer with them
 - We don't need any additional search system
+- No centralized infrastructure that could be compromised at a risk of whole communities being lost
 
 #### Disadvantages
 
 - New people can't find others, and others can't find them
-- Networks may be disjoint (i.e. two or more disconnected groups), and therefore not know about each other, essentially making separate communities
-- Offers have to be public. Currently Solid doesn't have any mechanism to give permissions to friends of friends of friends of friends, and nobody else. (We could make a crawler that collects these "foafoafoafs" to a single group, and give permissions to that group.)
-- We don't have any notion of a Community (e.g. cyclists, hitchhikers, ...) - like [different](https://welcometomygarden.org/) [hospex](https://www.bewelcome.org/) [communities](https://www.facebook.com/groups/hostasister/), or different [Trustroots Circles](https://www.trustroots.org/circles)
+- Offers have to be public. Currently, Solid doesn't have any mechanism to give permissions to friends of friends of friends of friends, and nobody else. (We could make a crawler that for each user collects these "foafoafoafs" to a single group, traversing the graph to certain depth, and give permissions to that group (or rather request members of that group to give permission to read their data).)
+- We don't have any notion of a Community (e.g. cyclists, hitchhikers, ...) - like [different](https://welcometomygarden.org/) [hospex](https://www.bewelcome.org/) [communities](https://www.facebook.com/groups/hostasister/), or different [Trustroots Circles](https://www.trustroots.org/circles) (could be done with "tags" stored in a profile)
+- Search by location would require prior fetch and scan of a whole network of foafoafoafs, that would take waaaaaaay to long to be usable (scalability and app responsiveness issues) unless we cache that data locally but then it can get stale, needs refreshments, ... gets a bit complicated. A cache could be structured as a geospecial index though, specifically created for each user from their own network. That might be not that bad idea, actually.
+- There is no common space to broadcast messages (invitations to stay or requests to be hosted), unless we had a wall of messages from foafoafoafs, stored locally and refreshed similarly as messages in SSB?
```

Comment:
I think this already assumes an architecture: That we don't send messages to others directly, but put them on a wall for others to discover?
Or is this about messages that are public by their nature? Like request for an area (not a person)?

I think this, also with comparing to SSB, makes it a bit complicated. I'd keep it for future reference, but prefer not to include for now, for simplicity and clarity.

Definitely makes sense to include when we'll start discussing messaging architecture (future)